composer.json

```
@@ -24,5 +24,8 @@
                 "Onliner\\Laravel\\CommandBus\\Providers\\CommandBusProvider"
             ]
         }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.5"
```

since php min version is 7.2, this should be like 
```suggestion
        "phpunit/phpunit": "^8.5 || ^9.5"
```